lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -436,7 +436,7 @@ protected function transformReportbackItem($data) {
    */
   protected function transformUser($data) {
     return array(
-      'id' => $data->uid,
+      'id' => isset($data->id) ? $data->id : $data->uid,
```

Comment:
So this could use an eventual refactor pass. Sometimes when the data is passed to the method it's property name is `id`, other times it's `uid`... it's the same situation that happened when transforming a Campaign:

https://github.com/DoSomething/dosomething/blob/dev/lib/modules/dosomething/dosomething_api/includes/Transformer.php#L237